deployment/terraform/create.go

```diff
@@ -222,25 +224,65 @@ func (t *Terraform) Create(initData bool) error {
 		}
 	}
 
-	// Note: This MUST be done after app servers have been set up.
-	// Otherwise, the vacuuming command will fail because no tables would
-	// have been created by then.
-	if t.output.HasDB() {
-		// Load the dump if specified
-		if !initData && t.config.DBDumpURI != "" {
-			err = t.IngestDump()
+	// Ingesting the DB dump and restoring the Elasticsearch snapshot can
+	// happen concurrently to reduce the deployment's total time
```

Comment:
Just to clarify: this will only benefit when we are using the gzip dump right? For the AWS backup approach, this does not make any difference.

Reply:
Correct.